tests/Profile.spec.ts

```diff
@@ -60,27 +82,35 @@ test.describe(`Testfälle für das eigene Profil anzeigen: Umgebung: ${process.e
       await expect(ProfileView.label_VornameNachname).toHaveText('Vor- und Nachname:');
       await expect(ProfileView.data_VornameNachname).toHaveText(Vorname + ' ' + Nachname);
       await expect(ProfileView.label_Benutzername).toHaveText('Benutzername:');
-      await expect(ProfileView.data_Benutzername).toHaveText(Benutzername);
-      await expect(ProfileView.label_KopersNr).toHaveText('KoPers.-Nr.:');
-      await expect(ProfileView.data_KopersNr).toHaveText(KopersNr);
+      await expect(ProfileView.data_Benutzername).toHaveText(benutzername);
+      await expect(ProfileView.label_KopersNr).toBeHidden();
+      await expect(ProfileView.data_KopersNr).toBeHidden();
       await expect(ProfileView.icon_InfoPersoenlicheDaten).toBeVisible();
-      // Schulzuordnung  
-      // Kann wegen dem bug: SPSH-1085 nicht getestet werden 
-      // await expect(ProfileView.cardHeadline_Schulzuordnung1).toHaveText('Schulzuordnung');
-      // await expect(ProfileView.label_Schule1).toHaveText('Schule:');
-      // await expect(ProfileView.data_Schule1).toHaveText(Organisation);
-      // await expect(ProfileView.label_Rolle1).toHaveText('Rolle:');
-      // await expect(ProfileView.data_Rolle1).toHaveText(Rolle);
-      // await expect(ProfileView.label_Dienststellennummer1).toHaveText('DStNr.:');
-      // await expect(ProfileView.data_Dienststellennummer1).toHaveText(Dienststellennummer);
+      // Schulzuordnung   
+      await expect(ProfileView.cardHeadline_Schulzuordnung1).toHaveText('Schulzuordnung');
+      await expect(ProfileView.label_Schule1).toHaveText('Schule:');
+      await expect(ProfileView.data_Schule1).toHaveText(Organisation);
+      await expect(ProfileView.label_Rolle1).toHaveText('Rolle:');
+      await expect(ProfileView.data_Rolle1).toHaveText(Rollenname);
+      await expect(ProfileView.label_Dienststellennummer1).toBeHidden();
+      await expect(ProfileView.data_Dienststellennummer1).toBeHidden();
       // Passwort
       await expect(ProfileView.cardHeadline_Passwort).toHaveText('Passwort');
       await expect(ProfileView.icon_Schluessel_Passwort).toBeVisible();
       await expect(ProfileView.button_NeuesPasswortSetzen).toBeEnabled();
       // 2FA
       await expect(ProfileView.cardHeadline_2FA).toHaveText('Zwei-Faktor-Authentifizierung');
       await expect(ProfileView.icon_Schild2FA).toBeVisible();
-      // await expect(ProfileView.button_2FAEinrichten).toBeDisabled(); Aktuell disabled im FE bis SPSH-855 fertig ist
+      await expect(ProfileView.text_no2FA).toHaveText('Es wurde noch kein zweiter Faktor für Sie eingerichtet.');
+      await expect(ProfileView.button_2FAEinrichten).toBeEnabled();
+    });
+
+    await test.step(`Testdaten via api löschen`, async () => {
```

Comment:
Wenn dir in der Mitte der Test abraucht, bleiben die Daten aber hängen! Es wäre besser, das im afterEach zu machen und sich per flag zu merken ob die Anlage geklappt hat.

Reply:
Korrekt, das betrifft die anderen tests auch. Ich würde das gerne generell für alle Tests in einem seperatem branch ändern ; hier geht es darum rote Tests zu fixen.